tests/Api/ApiAdressesTest.php

```diff
@@ -131,4 +130,115 @@ public function test_address_get_addressid()
             'postal_code' => '12345',
         ]);
     }
+
+    public function test_address_post()
+    {
+        $user = $this->signin();
+        $contact = factory('App\Contact')->create(['account_id' => $user->account->id]);
+
+        $response = $this->json('POST', '/api/addresses', [
+            'account_id' => $user->account->id,
+            'contact_id' => $contact->id,
+            'name' => 'address name',
+            'street' => 'street',
+            'postal_code' => '12345',
+            'country' => 'FR'
+        ]);
+
+        $response->assertStatus(201);
+
+        $this->assertDatabaseHas('addresses', [
+            'account_id' => $user->account->id,
+            'contact_id' => $contact->id,
+            'name' => 'address name',
+            'street' => 'street',
+            'postal_code' => '12345',
+            'country' => 'FR'
+        ]);
+
+        $response->assertJsonFragment([
+            'object' => 'address',
+            'id' => 1,
```

Comment:
IDs shouldn't be hardcoded. In my locale environment, each time I run the tests, the ID increments, making this test fail everytime.

Reply:
@asbiin ping.

Reply:
Here you can't really test the ID number. You can test the presence of an ID though, or you could test that the table `addresses` contains no row, then one row after your update is complete.

Reply:
Good idea

Reply:
Fixed by #1333